google/data_source_google_compute_default_service_account.go

```diff
@@ -0,0 +1,29 @@
+package google
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGoogleComputeDefaultServiceAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleComputeDefaultServiceAccountRead,
```

Comment:
Could you add an optional project field to the schema? This way, users can find the default service account for projects that are not necessarily set as their default Terraform provider project.

Reply:
I think it's ready for another pass